backend/src/main/java/com/woowacourse/gongcheck/application/GuestAuthService.java

```diff
@@ -13,9 +13,9 @@
 public class GuestAuthService {
 
     private final MemberRepository memberRepository;
-    private final JwtTokenProvider jwtTokenProvider;
+    private final JjwtTokenProvider jwtTokenProvider;
```

Comment:
```suggestion
    private final JwtTokenProvider jwtTokenProvider;
```
의존 받아야할 대상이 구현체로 잘못되어있네요 :)